src/routes/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { Suspense, lazy, ElementType } from 'react';
-import { Navigate, useRoutes, useLocation } from 'react-router-dom';
+import { Navigate, useLocation, Routes, Route } from 'react-router-dom';
```

Comment:
I agree it is simpler this way, but this is not an urgent change nor it is necessary at this point. Our main goals are successfully ported app into this template that works in dev and prod.

Reply:
Fine by me.